pkg/api/types.go

```diff
@@ -1380,3 +1381,15 @@ const (
 
 	PortHeader = "port"
 )
+
+// ResourceStats represents compute resource usage metrics associated with a pod or a container.
+type ResourceStats struct {
+	// Detailed instantaneous metrics.
+	Detailed *cadvisor_api.ContainerInfo `json:"detailed,omitempty"`
```

Comment:
FYI: Please ping me directly in the future. Mentioning me on github is no longer sufficient to rise about the noise.

Will discuss offline.